scripts/devenv-builder/configure-vm.sh

```diff
@@ -23,6 +25,7 @@ function usage() {
     echo "  --force-firewall          Install and configure firewalld regardless of other options"
     echo "  --no-set-release-version  Do NOT set the release subscription to the current release version"
     echo "  --skip-dnf-update         Do NOT run dnf update"
+    echo "  --pre-pull-images         Force pulling of container images before starting MicroShift"
```

Comment:
Maybe, just `--pull-images` without `pre`? 
Sorry for the nit, feel free to ignore it.

Reply:
Done